scrabble-site.rb

```diff
@@ -0,0 +1,30 @@
+require "sinatra"
+require "sinatra/reloader"
+require_relative "./lib/score.rb"
+require_relative "./lib/multi_word.rb"
+
+class ScrabbleSite < Sinatra::Base
```

Comment:
Really awesome job with this - This class is using appropriately named routes, appropriate HTTP verbs and no extraneous logic. Great great great.